tests/requirements.txt

```diff
@@ -0,0 +1 @@
+-e ../
```

Comment:
What about ppl who would like to run pytest. Currently now the flow is like create venv for ocs-ci like 
```bash
python3 -m venv .venv
source .venv/bin/activate
pip install -r requirements.txt
python run.py ......
```

If I would like to run pytest now, I don't have defined the pytest requirement anywhere just for tox.
So if we will stick with this project, I would have one place for all the requirements in setup.py as everything will be binded so closely with whole framework I think.  Or are you tending to have separate requirement file for tests requirement, so once ppl will need to have some package to be installed for their test that it should be defined in this file and not in the setup.py?  If so I still think that at least pytest and it's plugins should be in the setup.py.

Also as the user if I would like to install this requirements to the same venv as I have for ocs-ci I cannot do something like this from root directory of the project 
```bash
pip install -r tests/requirements.txt
Directory '../' is not installable. File 'setup.py' not found.
```
I have to `cd tests` and run `pip install -r requirements.txt`.

So please document a bit how this should be used for the users.

Reply:
+1 I don't see any value in using tox here to run ocsqe test cases.

Reply:
I've added ``pytest`` to setup.py and added a comment below about tox usage.